app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -662,7 +663,13 @@ void testAutomatedClaimFullPositivePresumptive() {
    */
   @Test
   void testAutomatedClaimFullPositiveIncompleteBloodPressures() {
-    testAutomatedClaimFullPositive("380");
```

Comment:
This is good but I hate to remove other checks in testAutomatedClaimFullPositive. Please take a look at what I did in https://github.com/department-of-veterans-affairs/abd-vro/pull/1362 and maybe return pdf from testAutomatedClaimFullPositive instead of startAutomatedClaim. Also I would another one for "-/50" from LH. And I would make sure there are no "-/-"s. Some version of Warren's fixes would have that issue earlier.